packages/eslint-plugin-office-addins/src/rules/no-empty-load.ts

```diff
@@ -27,11 +27,7 @@ export = {
   },
   create: function (context: any) {
     function isEmptyLoad(node: TSESTree.MemberExpression): boolean {
-      return (
-        isLoadFunction(node) &&
-        node.parent?.type == TSESTree.AST_NODE_TYPES.CallExpression &&
-        node.parent.arguments.length === 0
-      );
+      return isLoadFunction(node) && getLoadArgument(node) === "";
     }
```

Comment:
Why are we using getLoadArgument instead the other two test cases (which seem more to the point of the rule)?  Wouldn't it be simpler to just add the 'getLoadArgument(node) === ""' and an '||' with the length test (in parenthesis)?

Reply:
It was like that before, I just think it is better to reutilize the functions we already have, given that getLoadArgument is doing this now.

Reply:
It was doing the check for call expression before and we still had it here . . . though we weren't checking for the empty string before.

I guess if we are updating the meaning of "empty load" to include empty string then this could make sense.

Reply:
Updating to include empty strings I guess makes sense, as it is the same error and same problem (of performance)